extras/apt-example.list

```diff
@@ -2,6 +2,6 @@
 # See /etc/apt/sources.list.d/ for examples.
 
 # non-free sources
-#deb tor+http://ftp.us.debian.org/debian/ buster non-free
-#deb tor+http://ftp.us.debian.org/debian/ buster-backports non-free
-#deb tor+http://security.debian.org/ buster/updates non-free
+#deb tor+https://ftp.us.debian.org/debian/ buster non-free
+#deb tor+https://ftp.us.debian.org/debian/ buster-backports non-free
+#deb tor+https://security.debian.org/ buster/updates non-free
```

Comment:
The changes in this file don't work because those mirrors don't seem to have valid certificates. So apt-get fails trying to verify the hostnames. Those mirrors were taken from the Tails documentation and it looks like Tails uses http for their sources in /etc/apt as well (maybe for the same reason?). Anyway, unless you can think of a work around or mirrors that work with https for this, we have to continue to use http here. Otherwise HiddenVM will fail if the user decides to use the sources from this example file in his custom 'extras/apt.list' file, and HiddenVM decides to run 'apt-get update'.